src/utils/filters.js

```diff
@@ -22,25 +22,25 @@ export const filterByCategory = (products, categoryId) => {
  * @returns Returns all products with the brand selected
  */
 export const filterBySearch = (products, search) => {
-	return products.filter(
-		({ name, description }) =>
-			name.toLowerCase().includes(search.toLowerCase()) ||
-			description.toLowerCase().includes(search.toLowerCase())
-	);
+  return products.filter(
+    ({ name, description }) =>
+      name.toLowerCase().includes(search.toLowerCase()) ||
+      description.toLowerCase().includes(search.toLowerCase())
+  );
 };
 
 /**
  * @param {Array<{categories: {section_id: String}}>} products
  * @returns Returns all products with the brand selected
  */
 export const filterGeneric = (products) => {
-	return products.filter((product) => product.categories.section_id == "2");
+  return products.filter((product) => product.categories.section_id === "2");
```

Comment:
section_id is a String or a Number? I think this can cause problems if section_id return a string